src/execution.jl

```diff
@@ -203,8 +220,14 @@ macro benchmarkable(args...)
     end
     deleteat!(params, delinds)
 
+    quote_vars = Expr[]
+    core = quasiquote!(core, quote_vars)
```

Comment:
The test failure comes from catching the possible (though extreme) case where `core` here is a `Symbol`.